newIDE/app/src/EventsSheet/index.js

```diff
@@ -777,6 +788,35 @@ export default class EventsSheet extends React.Component<Props, State> {
     this.deleteSelection({ deleteInstructions: false });
   };
 
+  _replaceSelectionByGroupOfEvents = (eventsList: gdEventsList) => {
+    const contexts = getSelectedEventContexts(this.state.selection);
+    if (!contexts.length) return;
+
+    const newEvents = this.addNewEvent(
+      'BuiltinCommonInstructions::Group',
+      contexts[0]
+    );
+    if (!newEvents.length) {
+      console.error('A new event should have been created');
+      return;
+    }
+
+    const groupEvent = gd.asGroupEvent(newEvents[0]);
+
+    groupEvent.setName('Grouped events');
+    groupEvent.setFolded(true);
+    groupEvent
+      .getSubEvents()
+      .insertEvents(
+        eventsList,
+        0,
+        eventsList.getEventsCount(),
+        groupEvent.indexInList
```

Comment:
groupEvent is a `gd.GroupEvent`: https://github.com/4ian/GDevelop/blob/master/GDevelop.js/Bindings/Bindings.idl#L1386-L1416

It does not have a indexInList property - which is something that is only on objects created to represent a selection. This will evaluate to undefined, which, luckily, will be converted to 0 in C++ ;)

insertEvents means: "insert this list called `eventsList`, from event number `0` to number `eventsList.getEventsCount()` (so the whole list) in `groupEvent` subevents, at the position indicated by the last parameter".

I'll modify this to be simply 0 ;) (whatever you indicate, the group event subevents are empty, so any number will work.